helper/src/configpresets/baselines.json

```diff
@@ -20,51 +20,78 @@
-                                    "description": "Azure Container Registry (public)"
+                                    "description": "Azure Firewall for outbound traffic",
+                                    "linksrc": "https://learn.microsoft.com/en-us/azure/architecture/example-scenario/aks-firewall/aks-firewall"
+                                },
+                                {
+                                    "description": "Traefik Ingress Controller"
+                                },
+                                {
+                                    "description": "Azure Container Registry (private)"
+                                },
+                                {
+                                    "description": "Restrict dependencies with Private Link",
+                                    "linksrc": "https://docs.microsoft.com/en-us/azure/private-link/private-link-overview"
                                 },
                                 {
                                     "description": "AAD Integration",
                                     "linksrc": "https://docs.microsoft.com/en-gb/azure/aks/managed-aad"
                                 },
                                 {
-                                    "description": "Audit Pod security baseline standards",
+                                    "description": "Audit restricted pod security standards",
                                     "linksrc": "https://docs.microsoft.com/en-us/azure/governance/policy/concepts/policy-for-kubernetes"
+                                },
+                                {
+                                    "description": "East-West traffic control",
+                                    "linksrc": "https://docs.microsoft.com/en-gb/azure/aks/use-network-policies"
+                                },
+                                {
+                                    "description": "Store Kubernetes Secrets in Azure KeyVault,",
+                                    "linksrc": "https://docs.microsoft.com/en-us/azure/aks/csi-secrets-store-driver"
                                 }
                             ]
                         },
                         "imageSrc": "https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks/images/secure-baseline-architecture.svg",
                         "values": {
                             "cluster": {
                                 "enable_aad": true,
-                                "AksDisableLocalAccounts": false,
+                                "AksDisableLocalAccounts": true,
                                 "apisecurity": "none",
                                 "autoscale": true,
                                 "agentCount": 2,
-                                "maxCount": 10,
+                                "maxCount": 5,
                                 "upgradeChannel": "none",
```

Comment:
The baseline uses `Stable`

https://github.com/mspnp/aks-baseline/blob/main/cluster-stamp.bicep#L72